psi4/driver/schema_wrapper.py

```diff
@@ -87,7 +87,7 @@
     "scf": {
         "scf_one_electron_energy": {"variables": "ONE-ELECTRON ENERGY"},
         "scf_two_electron_energy": {"variables": "TWO-ELECTRON ENERGY"},
-        "scf_dipole_moment": {"variables": ["SCF DIPOLE X", "SCF DIPOLE Y", "SCF DIPOLE Z"], "conversion_factor": (1 / qcel.constants.dipmom_au2debye)},
+        "scf_dipole_moment": {"variables": "SCF DIPOLE"},
```

Comment:
Just out of interest, why not uncomment the quadrupole moment lines below? Or is it not part of the schema, yet?

Reply:
quadrupole isn't in the schema yet. but the real reason is add the `mtd +` bit to the qcvar string in the exceptions section https://github.com/psi4/psi4/pull/1800/files#diff-a54f9d4707cbe64f08c3df0dfc0c8376R567

Reply:
OK maybe this is a stupid question, but if there's an "SCF DIPOLE" why isn't there a "SCF QUADRUPOLE"?

Reply:
good question. qcvar-wise, there now is, as of this PR. lhs-wise, that's dictated by qcschema, and I've just petitioned for a first quadrupole, https://github.com/MolSSI/QCElemental/pull/207/files#diff-f178293574dc070baaac6713e52806beR54 . probably, no one had made a decision about complete/compressed and the shape of the array. that is, `(6,)` vs. `(3, 3)` for quadrupole.